### ISSUE TITLE
Interaction to create / delete memos.

### ISSUE THREAD

**Issue by jonathontoon** [jonathontoon/manifest#3]
## Expected Behavior
Some kind of UI affordance or keyboard shortcut (preferred IMO) to create or delete a memo.

## Current Behavior
Nothing exists. 😢 

## Possible Solution
 - Draw coordinates on the screen in an empty space with the mouse and a new memo will appear.
- Press "n" on the keyboard and a new memo will appear at a preset area.
- Press "d" on the current active memo and it will be deleted (after a prompt confirms the action).
- Some kind of toolbar where crucial actions could be housed.

## Context
Basically memgrid is unusable until we add this.

**Comment by jonathontoon** [comment]
Creation mode works fairly well. But after using "n" as the trigger to turn into creation mode, I'm thinking that simply clicking on the board (which no memo selection) should initial the mode. Pressing ESC would cancel.

**Comment by jonathontoon** [comment]
I think adding a small bit of UI for delete will help make things less ambiguous.
